apps/examples/dtls/dtls_server/Make.defs

```diff
@@ -17,5 +17,5 @@
 ###########################################################################
 
 ifeq ($(CONFIG_EXAMPLES_DTLS_SERVER),y)
-CONFIGURED_APPS += examples/dtls_server
+CONFIGURED_APPS += examples/dtls/dtls_server
```

Comment:
How about `dtls/client` and `dtls/server`?

Reply:
I agree with your opinion. I updated